foundation-extension/4-projects/business_unit_1/non-production/boa_gke.tf

```diff
@@ -15,13 +15,26 @@
  */
 
 locals {
-  gsa_sa_roles = {
+  boa_gsa_roles = {
     gke = [
       "roles/cloudtrace.agent",
       "roles/monitoring.metricWriter"
     ],
     sql = [
-      "roles/cloudsql.client"
+      "roles/cloudsql.admin"
```

Comment:
```suggestion
      "roles/cloudsql.admin" // TODO: reduce permissions 
```

Reply:
This cannot be reduced it was already in the reduced state, but the API call made my the BOA App is exactly for SQL Admin and fails if not given can be verified with Dima

Reply:
I believe it can be reduced we can add this comment for now and I will follow up with Dima.

 The app only requires read/write access to DB where as CSQL admin gives full control for all Cloud SQL resources including deleting which would be bad incase an application pod is compromised. Generally, this should just require the client role (connect to db) and then the ACLs are managed natively by the DB itself.